### PR TITLE
Update json behavior to be 9.3 compatible

### PR DESCRIPTION
**Pull request by rmarianski**
Connects to https://github.com/mapzen/vector-datasource/issues/826

@zerebubuth could you review please?

### CODE REVIEW

File: queries/earth.jinja2 [hunk 1]
```diff
@@ -21,14 +21,14 @@ FROM
 WHERE
     mz_earth_min_zoom <= {{ zoom }} AND
     {{ bounds|bbox_filter('the_geom') }}
-GROUP BY mz_properties
+GROUP BY gid
```
Comment (zerebubuth) [inline queries/earth.jinja2]: Is it not possible to group by JSON data types?

Comment (rmarianski) [inline queries/earth.jinja2]: Alas no, postgresql complained about it.

If it's safe to assume that we always have a `kind`, would it be better to group on that?

Comment (rmarianski) [inline queries/earth.jinja2]: Or maybe we should coerce the json to a string?

Comment (zerebubuth) [inline queries/earth.jinja2]: Actually, I'm getting errors like:

```
ProgrammingError: column "t.*" must appear in the GROUP BY clause or be used in an aggregate function
LINE 6:     mz_calculate_json_earth(t.*) AS mz_properties,
                                    ^
```

Comment (rmarianski) [inline queries/earth.jinja2]: Hhmm, I missed the updates in the `water` queries and am seeing that too.

Comment (rmarianski) [inline queries/earth.jinja2]: I eliminated the `group by`s in 24c9a4e.

**Comment by zerebubuth** [comment]
:+1: Thanks! Let's take a look at it on dev

**Pull request merged**